util.go

```diff
@@ -37,12 +38,14 @@ func writeFile(reader io.Reader, path string) error {
 
 	file, err := os.Create(path)
 	if err != nil {
-		return err
+		return errors.Wrap(err, "problem creating file")
 	}
-
 	if _, err := io.Copy(file, reader); err != nil {
-		return err
+		catcher := grip.NewBasicCatcher()
+		catcher.Add(errors.Wrap(err, "problem writing file"))
```

Comment:
Could you create the catcher before `io.Copy`. This would allow you to only have to call `file.Close` once, since you could add errors from both `io.Copy` and `file.Close` to the catcher.

Reply:
Done.